src/analysis/processing/qgsalgorithmxyztiles.cpp

```diff
@@ -111,7 +111,7 @@ void QgsXyzTilesBaseAlgorithm::createCommonParameters()
   addParameter( new QgsProcessingParameterNumber( QStringLiteral( "DPI" ), QObject::tr( "DPI" ), QgsProcessingParameterNumber::Integer, 96, false, 48, 600 ) );
   addParameter( new QgsProcessingParameterColor( QStringLiteral( "BACKGROUND_COLOR" ), QObject::tr( "Background color" ), QColor( Qt::transparent ), true, true ) );
   addParameter( new QgsProcessingParameterBoolean( QStringLiteral( "ANTIALIAS" ), QObject::tr( "Enable antialiasing" ), true ) );
-  addParameter( new QgsProcessingParameterBoolean( QStringLiteral( "Skip Empty Tiles" ), QObject::tr( "Skip empty tiles" ), true ) );
+  addParameter( new QgsProcessingParameterBoolean( QStringLiteral( "Skip_Empty_Tiles" ), QObject::tr( "Skip empty tiles" ), true, true ) );
```

Comment:
```suggestion
  addParameter( new QgsProcessingParameterBoolean( QStringLiteral( "SKIP_EMPTY_TILES" ), QObject::tr( "Skip empty tiles" ), true, true ) );
```

let's be consistent with capitalization